js/payone/core/payolution.js

```diff
@@ -157,4 +157,23 @@ function switchInstallmentPlan(sKey, sCode, iInstallments)
     ];
     switchVisibility(aShow, true);
     $(sCode + '_selected_installmentplan').value = iInstallments;
+}
+
+/**
+ *
+ * @param payment_code
+ */
+function payonePayolutionCustomerDobInput(payment_code)
+{
+    var daySelect = $(payment_code + '_additional_fields_customer_dob_day');
+    var monthSelect = $(payment_code + '_additional_fields_customer_dob_month');
+    var yearSelect = $(payment_code + '_additional_fields_customer_dob_year');
+    var hiddenDobFull = $(payment_code + '_additional_fields_customer_dob_full');
+
+    if (daySelect == undefined || monthSelect == undefined || yearSelect == undefined
```

Comment:
A non-existing jQuery selector won't result in an undefined value. The return value will be a jQuery object which length is zero. The condition always evaluates to false.